README.md

```diff
@@ -8,6 +8,7 @@ A flexible, modern, cross-platform C++ recursive Bayesian estimation library.
 # Overview
 - [⚠️ About versioning](#️-about-versioning)
 - [📖 Background](#-background)
+- [Installing with conda](#installing-with-conda)
```

Comment:
I think we could add a small icon here to adhere with the style of the `README.md`.

Thank you